spanner/cloud-client/batch_sample.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to do batch operations using Cloud
+Spanner.
+
+For more information, see the README.rst under /spanner.
+"""
+
+import argparse
+from multiprocessing.pool import ThreadPool
```

Comment:
I'd suggest you use `concurrent.futures` which is a preferred over directly using `multiprocessing.pool`.

Reply:
Fixed in 172ee2c8c4a64f32f0713292153969e009854543.